developers/README.md

```diff
@@ -17,6 +17,44 @@ Supported development platforms:
 * [Ubuntu](dev-environment/ubuntu.md)
 * [Apple's macOS](dev-environment/macOS.md)
 
+### Virtual Machines
```

Comment:
What do you mean "links back to the Development Environment?" We can link across Markdown files pretty easily.

I'm basically saying the same thing we discussed on #22. I think we should have the general configuration information (linked above), and then specific lore for VMs and Docker.